src/google_calendar_tool.js

```diff
@@ -48,44 +36,62 @@ export class GoogleCalendarTool extends Tool {
 
     return auth
   }
+}
 
-  async runClassification(query) {
-    const prompt = new PromptTemplate({
-      template: EVENT_CLASSIFICATION_PROMPT,
-      inputVariables: ['query']
+export class GoogleCalendarCreateTool extends GoogleCalendarBase {
```

Comment:
Nit: can we move these new classes to their own files?

It's a good convention to follow and will help readability, review of changes etc

Reply:
I've pushed up a commit for this: https://github.com/nearform/langchain-google-calendar/pull/21/commits/9044160d45f02d2e2c2bd40137a3b3180fba747d